.github/workflows/integration.yml

```
@@ -38,6 +39,10 @@ jobs:
             CLIENT_USE_M2CRYPTO: Yes
           - SERVER_USE_M2CRYPTO: No
             CLIENT_USE_M2CRYPTO: No
+          # Fail deprecation
```

To be consistent with the other options:
```suggestion
          ###### Fail deprecation
```

---

done